recipe/patches/0002-Consolidated-build-fixes-for-XLA.patch

```diff
@@ -0,0 +1,417 @@
+From b7b618bc47586c9ad4b6bbd0113b67e4feb7e3ff Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Thu, 14 Dec 2023 17:06:15 +0100
+Subject: [PATCH 2/2] Consolidated build fixes for XLA
+
+---
```

Comment:
I think a bit more in the way of instructions how to maintain this would be helpful, something like
```suggestion
Subject: [PATCH 2/2] Consolidated build fixes for XLA

jax vendors xla, but only populates the sources through bazel, so we cannot
patch as usual through conda, but rather need to teach the bazel build file
for xla to apply those patches.

To maintain/rebase these patches, use a checkout of https://github.com/openxla/xla,
and then rebase to the commit of xla being used by jax, which can be found in
https://github.com/google/jax/blob/main/third_party/xla/workspace.bzl,
which is also where we're patching in the list of patches to apply.
---
```